core/domain/classifier_services.py

```diff
@@ -118,3 +122,74 @@ def classify_string_classifier_rule(state, normalized_answer):
             return None
 
     return None
+
+
+def get_classifier_from_model(classifier_model):
+    """Returns a classifier domian object given a classifier model loaded
+    from datastore.
+    """
+    return classifier_domain.Classifier(
+        classifier_model.id, classifier_model.exp_id,
+        classifier_model.exp_version_when_created,
+        classifier_model.state_name, classifier_model.algorithm_id,
+        classifier_model.cached_classifier_data,
+        classifier_model.data_schema_version)
+
+def get_classifier_by_id(classifier_id):
+    """Returns a classifier domain object given a classifier id.
+    """
+    classifier_model = classifier_models.ClassifierModel.get(
+        classifier_id, strict=False)
+    if classifier_model is None:
+        raise Exception("No classifier found for the classifer's id.")
+    else:
+        classifier = get_classifier_from_model(classifier_model)
+        return classifier
+
+
+def _create_classifier(classifier):
+    """Creates classifier model in the datastoer given a classifier
+       domain object.
+    """
+    classifier_models.ClassifierModel.create(
+        classifier.exp_id, classifier.exp_version_when_created,
+        classifier.state_name, classifier.algorithm_id,
+        classifier.cached_classifier_data, classifier.data_schema_version,
+        )
+
+
+def _save_classifier(classifier_model, classifier):
```

Comment:
@anmolshkl -- does exp_version_when_created need to update when the state name is updated (or do we need anything keeping track of the "current version"? I just wanted to check if we had any issues where someone tries to look up a given version and see if it contains the current state name, but that state name doesn't exist in that version and we've deleted the old state name.

[Note: edited for clarity.]